iroh-net/Cargo.toml

```diff
@@ -94,6 +95,7 @@ swarm-discovery = { version = "0.2.1", optional = true }
 
 # dht_discovery
 genawaiter = { version = "0.99.1", features = ["futures03"], optional = true }
+netwatch = { version = "0.1.0", path = "../net-tools/netwatch" }
```

Comment:
I think this needs to be up with portmapper?

Reply:
moved (and sorted) in [5131a5f](https://github.com/n0-computer/iroh/pull/2855/commits/5131a5fc2e75ad5012eab46fbe100d09b29d25ef)